### PR TITLE
Add documentation how to debug memory leaks

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: Contributor Documentation/Debugging Memory Leaks.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Debugging Memory Leaks
```
Comment (bnbarham) [inline Contributor Documentation/Debugging Memory Leaks.md:1]: Could add a link to eg. https://www.swift.org/documentation/server/guides/memory-leaks-and-usage.html with this being a TLDR?

**Review by bnbarham — APPROVED** [review]

File: Contributor Documentation/Debugging Memory Leaks.md (hunk 1 above)
Comment (ahoppen) [reply Contributor Documentation/Debugging Memory Leaks.md:1]: Updated

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Pull request merged**